packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.tsx

```diff
@@ -108,23 +131,36 @@ export class AtomicResultPrintableUri {
     ];
   }
 
+  private renderLink(content: VNode | string | null, uri: string) {
+    return (
+      <ResultLink
+        interactiveResult={this.interactiveResult}
+        href={uri}
```

Comment:
I don't see any `href` attribute on `ResultLink`, is it possible that all those ResultLinks are pointing to the same location (the clickUri)?

Also, `filterProtocol` is missing here, it should be added either here or in ResultLink.